test/jsonValidationTest.js

```diff
@@ -125,6 +127,16 @@ function validateJsonExistForWorkbook(rootPath, results, file) {
     });
 }
 
+function validateWorkbookExistForSettings(file) {
+    let folder = path.dirname(file);
+    fs.readdir(folder, (err, list) => {
+        let workbooks = list.filter(s => s.endsWith(".workbook"));
```

Comment:
do we still support cohort? if so, you'll need to check for .cohort

Reply:
true.  but nobody's touched cohorts in literal years, so i don't think its worth the time there.